errors in showcase

I encountered these errors when running `showcase`:

### LargeTable

```
panic: not yet implemented: you should implement `change` method for a component with properties
```

when it "kind of" works it seems to be very buggy -- adding buttons to other scenes.

### TwoApps
```
Panic error message: called `Option::unwrap()` on a `None` value
showcase.js:572:17
Panic location: libcore/option.rs:335
showcase.js:677:17
RuntimeError: unreachable executed
showcase.js:1919579:1
```

### GameOfLife

<details><summary>DETAILS</summary>
```
http://[::1]:8000/showcase.js:689 Encountered a panic!
__extjs_dc2fd915bd92f9e9c6a3bd15174f1414eee3dbaf @ http://[::1]:8000/showcase.js:689
stdweb::webcore::initialization::initialize::{{closure}}::h362950e78055cb41 @ wasm-0098e44a-2690:8
std::panicking::rust_panic_with_hook::h3b8543970eee8168 @ wasm-0098e44a-3395:121
std::panicking::begin_panic_fmt::h2b9c4c2ee63d50ac @ wasm-0098e44a-3394:94
rust_begin_unwind @ wasm-0098e44a-3393:48
core::panicking::panic_fmt::h4933d8af4ae31777 @ wasm-0098e44a-3742:26
core::panicking::panic::hf47217b874ee6a24 @ wasm-0098e44a-3741:46
<yew::html::Scope<CTX, COMP>>::mount_in_place::hd1ff6b09bd23eae1 @ wasm-0098e44a-1239:310
<yew::virtual_dom::vcomp::VComp<CTX, COMP>>::lazy::{{closure}}::h2f2d5572e74aded3 @ wasm-0098e44a-1915:124
<yew::virtual_dom::vcomp::VComp<CTX, COMP> as yew::virtual_dom::VDiff>::apply::h90afe3e1686b6571 @ wasm-0098e44a-1956:1356
<yew::virtual_dom::vnode::VNode<CTX, COMP> as yew::virtual_dom::VDiff>::apply::h65b0e3764e53c3d0 @ wasm-0098e44a-1200:277
<yew::virtual_dom::vtag::VTag<CTX, COMP> as yew::virtual_dom::VDiff>::apply::hc3ae17643be2e598 @ wasm-0098e44a-987:5065
<yew::virtual_dom::vnode::VNode<CTX, COMP> as yew::virtual_dom::VDiff>::apply::h65b0e3764e53c3d0 @ wasm-0098e44a-1200:147
<stdweb::webcore::newtype::Newtype<(stdweb::webcore::serialization::FunctionTag, ()), F> as stdweb::webcore::serialization::FuncallAdapter<F>>::funcall_adapter::hb45e364f17d51435 @ wasm-0098e44a-1038:774
Module.STDWEB_PRIVATE.dyncall @ http://[::1]:8000/showcase.js:390
output @ http://[::1]:8000/showcase.js:116
routine @ http://[::1]:8000/showcase.js:671
setTimeout (async)
schedule_routine @ http://[::1]:8000/showcase.js:671
schedule @ http://[::1]:8000/showcase.js:671
__extjs_26f8148e18c88824e484983521208bc0906de12f @ http://[::1]:8000/showcase.js:692
<yew::html::ScopeSender<CTX, COMP>>::send::h648ba57ffc647a06 @ wasm-0098e44a-1342:64
<stdweb::webcore::newtype::Newtype<(stdweb::webcore::serialization::FunctionTag, (A12,)), F> as stdweb::webcore::serialization::FuncallAdapter<F>>::funcall_adapter::h788dd706465c082c @ wasm-0098e44a-1151:271
Module.STDWEB_PRIVATE.dyncall @ http://[::1]:8000/showcase.js:390
output @ http://[::1]:8000/showcase.js:116
http://[::1]:8000/showcase.js:572 Panic error message: called `Option::unwrap()` on a `None` value
__extjs_97495987af1720d8a9a923fa4683a7b683e3acd6 @ http://[::1]:8000/showcase.js:572
stdweb::webcore::initialization::initialize::{{closure}}::h362950e78055cb41 @ wasm-0098e44a-2690:37
std::panicking::rust_panic_with_hook::h3b8543970eee8168 @ wasm-0098e44a-3395:121
std::panicking::begin_panic_fmt::h2b9c4c2ee63d50ac @ wasm-0098e44a-3394:94
rust_begin_unwind @ wasm-0098e44a-3393:48
core::panicking::panic_fmt::h4933d8af4ae31777 @ wasm-0098e44a-3742:26
core::panicking::panic::hf47217b874ee6a24 @ wasm-0098e44a-3741:46
<yew::html::Scope<CTX, COMP>>::mount_in_place::hd1ff6b09bd23eae1 @ wasm-0098e44a-1239:310
<yew::virtual_dom::vcomp::VComp<CTX, COMP>>::lazy::{{closure}}::h2f2d5572e74aded3 @ wasm-0098e44a-1915:124
<yew::virtual_dom::vcomp::VComp<CTX, COMP> as yew::virtual_dom::VDiff>::apply::h90afe3e1686b6571 @ wasm-0098e44a-1956:1356
<yew::virtual_dom::vnode::VNode<CTX, COMP> as yew::virtual_dom::VDiff>::apply::h65b0e3764e53c3d0 @ wasm-0098e44a-1200:277
<yew::virtual_dom::vtag::VTag<CTX, COMP> as yew::virtual_dom::VDiff>::apply::hc3ae17643be2e598 @ wasm-0098e44a-987:5065
<yew::virtual_dom::vnode::VNode<CTX, COMP> as yew::virtual_dom::VDiff>::apply::h65b0e3764e53c3d0 @ wasm-0098e44a-1200:147
<stdweb::webcore::newtype::Newtype<(stdweb::webcore::serialization::FunctionTag, ()), F> as stdweb::webcore::serialization::FuncallAdapter<F>>::funcall_adapter::hb45e364f17d51435 @ wasm-0098e44a-1038:774
Module.STDWEB_PRIVATE.dyncall @ http://[::1]:8000/showcase.js:390
output @ http://[::1]:8000/showcase.js:116
routine @ http://[::1]:8000/showcase.js:671
setTimeout (async)
schedule_routine @ http://[::1]:8000/showcase.js:671
schedule @ http://[::1]:8000/showcase.js:671
__extjs_26f8148e18c88824e484983521208bc0906de12f @ http://[::1]:8000/showcase.js:692
<yew::html::ScopeSender<CTX, COMP>>::send::h648ba57ffc647a06 @ wasm-0098e44a-1342:64
<stdweb::webcore::newtype::Newtype<(stdweb::webcore::serialization::FunctionTag, (A12,)), F> as stdweb::webcore::serialization::FuncallAdapter<F>>::funcall_adapter::h788dd706465c082c @ wasm-0098e44a-1151:271
Module.STDWEB_PRIVATE.dyncall @ http://[::1]:8000/showcase.js:390
output @ http://[::1]:8000/showcase.js:116
http://[::1]:8000/showcase.js:677 Panic location: libcore/option.rs:335
__extjs_72fc447820458c720c68d0d8e078ede631edd723 @ http://[::1]:8000/showcase.js:677
stdweb::webcore::initialization::initialize::{{closure}}::h362950e78055cb41 @ wasm-0098e44a-2690:55
std::panicking::rust_panic_with_hook::h3b8543970eee8168 @ wasm-0098e44a-3395:121
std::panicking::begin_panic_fmt::h2b9c4c2ee63d50ac @ wasm-0098e44a-3394:94
rust_begin_unwind @ wasm-0098e44a-3393:48
core::panicking::panic_fmt::h4933d8af4ae31777 @ wasm-0098e44a-3742:26
core::panicking::panic::hf47217b874ee6a24 @ wasm-0098e44a-3741:46
<yew::html::Scope<CTX, COMP>>::mount_in_place::hd1ff6b09bd23eae1 @ wasm-0098e44a-1239:310
<yew::virtual_dom::vcomp::VComp<CTX, COMP>>::lazy::{{closure}}::h2f2d5572e74aded3 @ wasm-0098e44a-1915:124
<yew::virtual_dom::vcomp::VComp<CTX, COMP> as yew::virtual_dom::VDiff>::apply::h90afe3e1686b6571 @ wasm-0098e44a-1956:1356
<yew::virtual_dom::vnode::VNode<CTX, COMP> as yew::virtual_dom::VDiff>::apply::h65b0e3764e53c3d0 @ wasm-0098e44a-1200:277
<yew::virtual_dom::vtag::VTag<CTX, COMP> as yew::virtual_dom::VDiff>::apply::hc3ae17643be2e598 @ wasm-0098e44a-987:5065
<yew::virtual_dom::vnode::VNode<CTX, COMP> as yew::virtual_dom::VDiff>::apply::h65b0e3764e53c3d0 @ wasm-0098e44a-1200:147
<stdweb::webcore::newtype::Newtype<(stdweb::webcore::serialization::FunctionTag, ()), F> as stdweb::webcore::serialization::FuncallAdapter<F>>::funcall_adapter::hb45e364f17d51435 @ wasm-0098e44a-1038:774
Module.STDWEB_PRIVATE.dyncall @ http://[::1]:8000/showcase.js:390
output @ http://[::1]:8000/showcase.js:116
routine @ http://[::1]:8000/showcase.js:671
setTimeout (async)
schedule_routine @ http://[::1]:8000/showcase.js:671
schedule @ http://[::1]:8000/showcase.js:671
__extjs_26f8148e18c88824e484983521208bc0906de12f @ http://[::1]:8000/showcase.js:692
<yew::html::ScopeSender<CTX, COMP>>::send::h648ba57ffc647a06 @ wasm-0098e44a-1342:64
<stdweb::webcore::newtype::Newtype<(stdweb::webcore::serialization::FunctionTag, (A12,)), F> as stdweb::webcore::serialization::FuncallAdapter<F>>::funcall_adapter::h788dd706465c082c @ wasm-0098e44a-1151:271
Module.STDWEB_PRIVATE.dyncall @ http://[::1]:8000/showcase.js:390
output @ http://[::1]:8000/showcase.js:116
wasm-0098e44a-3556:2 Uncaught RuntimeError: unreachable
    at __rust_start_panic (wasm-function[3556]:1)
    at rust_panic (wasm-function[3396]:18)
    at std::panicking::rust_panic_with_hook::h3b8543970eee8168 (wasm-function[3395]:346)
    at std::panicking::begin_panic_fmt::h2b9c4c2ee63d50ac (wasm-function[3394]:192)
    at rust_begin_unwind (wasm-function[3393]:98)
    at core::panicking::panic_fmt::h4933d8af4ae31777 (wasm-function[3742]:57)
    at core::panicking::panic::hf47217b874ee6a24 (wasm-function[3741]:99)
    at <yew::html::Scope<CTX, COMP>>::mount_in_place::hd1ff6b09bd23eae1 (wasm-function[1239]:650)
    at <yew::virtual_dom::vcomp::VComp<CTX, COMP>>::lazy::{{closure}}::h2f2d5572e74aded3 (wasm-function[1915]:271)
    at <yew::virtual_dom::vcomp::VComp<CTX, COMP> as yew::virtual_dom::VDiff>::apply::h90afe3e1686b6571 (wasm-function[1956]:2736)
```
</details>

### NpmAndRest
When getting "gravatar"
```
GETXHR
https://gravatar.com/205e460b479e2e5b48aec07710c08d50
[HTTP/2.0 302 Found 876ms]
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://gravatar.com/205e460b479e2e5b48aec07710c08d50. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing).
TypeError: NetworkError when attempting to fetch resource.
[Learn More]
```

--- a/examples/showcase/src/main.rs
+++ b/examples/showcase/src/main.rs
@@ -37,7 +37,7 @@ use strum_macros::{Display, EnumIter, EnumString};
 use textarea::Model as Textarea;
 use timer::Model as Timer;
 use todomvc::Model as Todomvc;
-use two_apps::Model as TwoApps;
+use two_apps::TwoModels as TwoApps;
 use yew::components::Select;
 use yew::{html, App, Component, ComponentLink, Html, ShouldRender};
 

--- a/examples/std_web/two_apps/src/lib.rs
+++ b/examples/std_web/two_apps/src/lib.rs
@@ -1,6 +1,8 @@
 #![recursion_limit = "256"]
 
-use yew::{html, Component, ComponentLink, Html, ShouldRender};
+use stdweb::web::IParentNode;
+use yew::{html, App, Component, ComponentLink, Html, ShouldRender};
+use yew::html::{Scope};
 
 pub struct Model {
     link: ComponentLink<Self>,
@@ -69,6 +71,50 @@ impl Component for Model {
                 <button onclick=self.link.callback(|_| Msg::SendToOpposite("Two".into()))>{ "Two" }</button>
                 <button onclick=self.link.callback(|_| Msg::SendToOpposite("Three".into()))>{ "Three" }</button>
                 <button onclick=self.link.callback(|_| Msg::SendToOpposite("Ping".into()))>{ "Ping" }</button>
+            </div>
+        }
+    }
+}
+
+fn mount_app(selector: &'static str, app: App<Model>) -> Scope<Model> {
+    let document = yew::utils::document();
+    let element = document.query_selector(selector).unwrap().unwrap();
+    app.mount(element)
+}
+
+pub struct TwoModels {}
+
+impl Component for TwoModels {
+    type Message = ();
+    type Properties = ();
+
+    fn create(_: Self::Properties, _: ComponentLink<Self>) -> Self {
+        TwoModels { }
+    }
+
+    fn mounted(&mut self) -> ShouldRender {
+        let first_app = App::new();
+        let second_app = App::new();
+        let to_first = mount_app(".first-app", first_app);
+        let to_second = mount_app(".second-app", second_app);
+        to_first.send_message(Msg::SetOpposite(to_second.clone()));
+        to_second.send_message(Msg::SetOpposite(to_first.clone()));
+        false
+    }
+
+    fn update(&mut self, _: Self::Message) -> ShouldRender {
+        false
+    }
+
+    fn change(&mut self, _: Self::Properties) -> ShouldRender {
+        false
+    }
+
+    fn view(&self) -> Html {
+        html! {
+            <div>
+                <div class="first-app"></div>
+                <div class="second-app"></div>
             </div>
         }
     }

--- a/examples/std_web/two_apps/src/main.rs
+++ b/examples/std_web/two_apps/src/main.rs
@@ -15,7 +15,7 @@ fn main() {
     let second_app = App::new();
     let to_first = mount_app(".first-app", first_app);
     let to_second = mount_app(".second-app", second_app);
-    to_first.send_message(Msg::SetScope(to_second.clone()));
-    to_second.send_message(Msg::SetScope(to_first.clone()));
+    to_first.send_message(Msg::SetOpposite(to_second.clone()));
+    to_second.send_message(Msg::SetOpposite(to_first.clone()));
     yew::run_loop();
 }

--- a/examples/web_sys/two_apps/src/lib.rs
+++ b/examples/web_sys/two_apps/src/lib.rs
@@ -1,6 +1,7 @@
 #![recursion_limit = "256"]
 
-use yew::{html, Component, ComponentLink, Html, ShouldRender};
+use yew::{html, App, Component, ComponentLink, Html, ShouldRender};
+use yew::html::{Scope};
 
 pub struct Model {
     link: ComponentLink<Self>,
@@ -69,6 +70,50 @@ impl Component for Model {
                 <button onclick=self.link.callback(|_| Msg::SendToOpposite("Two".into()))>{ "Two" }</button>
                 <button onclick=self.link.callback(|_| Msg::SendToOpposite("Three".into()))>{ "Three" }</button>
                 <button onclick=self.link.callback(|_| Msg::SendToOpposite("Ping".into()))>{ "Ping" }</button>
+            </div>
+        }
+    }
+}
+
+fn mount_app(selector: &'static str, app: App<Model>) -> Scope<Model> {
+    let document = yew::utils::document();
+    let element = document.query_selector(selector).unwrap().unwrap();
+    app.mount(element)
+}
+
+pub struct TwoModels {}
+
+impl Component for TwoModels {
+    type Message = ();
+    type Properties = ();
+
+    fn create(_: Self::Properties, _: ComponentLink<Self>) -> Self {
+        TwoModels { }
+    }
+
+    fn mounted(&mut self) -> ShouldRender {
+        let first_app = App::new();
+        let second_app = App::new();
+        let to_first = mount_app(".first-app", first_app);
+        let to_second = mount_app(".second-app", second_app);
+        to_first.send_message(Msg::SetOpposite(to_second.clone()));
+        to_second.send_message(Msg::SetOpposite(to_first.clone()));
+        false
+    }
+
+    fn update(&mut self, _: Self::Message) -> ShouldRender {
+        false
+    }
+
+    fn change(&mut self, _: Self::Properties) -> ShouldRender {
+        false
+    }
+
+    fn view(&self) -> Html {
+        html! {
+            <div>
+                <div class="first-app"></div>
+                <div class="second-app"></div>
             </div>
         }
     }

--- a/examples/web_sys/two_apps/src/main.rs
+++ b/examples/web_sys/two_apps/src/main.rs
@@ -14,7 +14,7 @@ fn main() {
     let second_app = App::new();
     let to_first = mount_app(".first-app", first_app);
     let to_second = mount_app(".second-app", second_app);
-    to_first.send_message(Msg::SetScope(to_second.clone()));
-    to_second.send_message(Msg::SetScope(to_first.clone()));
+    to_first.send_message(Msg::SetOpposite(to_second.clone()));
+    to_second.send_message(Msg::SetOpposite(to_first.clone()));
     yew::run_loop();
 }

